pallets/asset-manager/Cargo.toml

```diff
@@ -9,24 +9,24 @@ version = "3.2.1"
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = '3.1.2', default-features = false }
-frame-benchmarking = { git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.22", default-features = false, optional = true }
-frame-support = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.22", default-features = false }
-frame-system = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.22", default-features = false }
+frame-benchmarking = { git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.26", default-features = false, optional = true }
+frame-support = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
+frame-system = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
 log = { version = "0.4.0", default-features = false }
 manta-primitives = { path = "../../primitives/manta", default-features = false }
 scale-info = { version = "2.0.0", default-features = false, features = ["derive"] }
-sp-runtime = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.22", default-features = false }
-sp-std = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.22", default-features = false }
-xcm = { git = 'https://github.com/paritytech/polkadot.git', default-features = false, branch = "release-v0.9.22" }
+sp-runtime = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
+sp-std = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
+xcm = { git = 'https://github.com/paritytech/polkadot.git', default-features = false, branch = "release-v0.9.26" }
 
 # 3rd party dependencies
-orml-traits = { git = "https://github.com/manta-network/open-runtime-module-library.git", default-features = false, rev = "cc26ffa" }
+orml-traits = { git = "https://github.com/open-web3-stack/open-runtime-module-library.git", default-features = false, branch = "polkadot-v0.9.26" }
```

Comment:
Not sure why to use our own fork.

Reply:
Vendoring. We can't trust ORML to outlive us

Reply:
Actually my bad  I can revert to our own fork, the main thing here was to use branch instead of commit hash.
But I feel like we should try to use the main repo unless there's a need to use our own fork like Nimbus `garandor/polkadot-v0.9.26` branch.

Reply:
i think even if we didnt need to customize we should still vendor deps that are non-parity.
So we don't depend on the ORML github to always exist

Reply:
Ok, if you guys want it that way, changing it.

Reply:
Btw if you guys ever wonder how to fully sync our forks with upstream repo and branches https://stackoverflow.com/a/50220475